### ISSUE TITLE
Memory leak during loading a model

### ISSUE THREAD

**Issue by gr3w2** [tensorflow/tfjs#1146]
#### TensorFlow.js version
@tensorflow/tfjs: 0.14.2,
@tensorflow/tfjs-node: 0.2.3
#### Browser version
node v11.7.0
#### Describe the problem or feature request
Hi there!
I found that method `loadModel` creates extra tensor variables, and there is no access to them, so they can't be disposed directly. I can dispose them using `disposeVariables`, but that function disposes all variables from memory. So if I train another model at the same time when I call `disposeVariables`, then the training will throw an error, 'cause it also use some variables inside and all of them was removed.
I attached a link to GitHub repo, so you can find a code to reproduce that behavior there
#### Code to reproduce the bug / link to feature request
https://github.com/gr3w2/tfjs-bug-report

**Comment by caisq** [comment]
@gr3w2 May I inquire as to what problems the small number of extra tensors created cause to you? In other words, why do you need to get to a state of 0 WebGL tensors in your use case? 

Also, does your model involve dropout layers? There is a recent PR that has fixed a memory leak issue in that particular layer type: https://github.com/tensorflow/tfjs-layers/pull/428 

There might be similar "leak" issues in other layer types, which we can fix once we know what they are.
